### PR TITLE
Mobile version project

### PR DESCRIPTION
**Pull request by mwanawabangona**
- Add Html
- Css
- Linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+
+  <body>
+    
+    <!-- POPUP BEGINNING PROGRAMMING
```
Comment (afanimartin) [inline index.html]: Please remove all comments from your codebase. Comments are ugly when left in the codebase.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,453 @@
...
+            <a href="#"><img src="./images/GitHub.svg" alt="GitHub" /></a>
+          </li>
+          <li>
+            <a href="#"><img src="./images/Vector.svg" alt="Vector" /></a>
+          </li>
+        </ul>
+      </div>
+    </header>
+
+    <section class="work" id="work">
```
Comment (afanimartin) [inline index.html]: It's a better idea to style this section with display of grid. Grid will enable the items to respond to any screen size hence making the page responsive.  Please make that change accordingly.

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @mwanawabangona, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (mwanawabangona) [reply index.html]: thank you will do just that

File: index.html (hunk 1 above)
Comment (mwanawabangona) [reply index.html]: Ok just did

**Review by Leon-Mbegera — APPROVED** [review]
Hi @mwanawabangona , 

# Approved ✔

- Well done for implementing all the requested changes 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it and move forward to the next stage. :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**